src/main/java/mate/academy/bookstore/repository/RoleRepository.java

```diff
@@ -0,0 +1,8 @@
+package mate.academy.bookstore.repository;
+
+import mate.academy.bookstore.model.Role;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface RoleRepository extends JpaRepository<Role, Long> {
+    Role getByName(Role.RoleName roleName);
```

Comment:
```suggestion
    Optional<Role> getByName(Role.RoleName roleName);
```

Reply:
Bohdan said earlier that there is a convention, if the method starts with `find' - to return Optional, and if `get' - Entity